cs.c

```diff
@@ -110,7 +110,7 @@ cs_vsnprintf_t cs_vsnprintf = NULL;
 #endif
 
 CAPSTONE_EXPORT
-unsigned int cs_version(int *major, int *minor)
+unsigned int CAPSTONE_API cs_version(int *major, int *minor)
```

Comment:
to be consistent, what do you think about moving CAPSTONE_API after CAPSTONE_EXPORT, so there is no separation between function type & function name?

so it will be like this:

```
CAPSTONE_EXPORT CAPSTONE_API
unsigned int cs_version(int *major, int *minor)
```